docker-compose.yml

```
@@ -0,0 +1,117 @@
+  redis:
+    image: library/redis:7.0-alpine
+    ports:
+      - 6379:6379
+    restart: unless-stopped
+    networks:
+      - saleor-backend-tier
+    volumes:
+      - saleor-redis:/data
+
+  worker:
+    build: . 
```

yamllint

[error] trailing spaces

(trailing-spaces)